src/lib/ProductExporter.ts

```diff
@@ -477,26 +477,31 @@ export class ProductExporter {
             ...chargeElements,
             ...chargeTiers
         ]);
-      
         for(let charge of charges){
-
-            let chargeName = charge['Name'];
-            let chargeToSave: any = {};
-            let chargeElementsToSave: any = [];
-            let chargeTiersToSave: any = [];
-            chargeToSave.root= charge;
             
-            chargeElements.filter(chargeElement => chargeElement['enxCPQ__Charge_Parent__r'] 
-                                                   && chargeElement['enxCPQ__Charge_Parent__r']['enxCPQ__TECH_External_Id__c'] === charge['enxCPQ__TECH_External_Id__c'])
-                          .forEach(chargeElementToSave=>{ chargeElementsToSave.push(chargeElementToSave)});
-
-            chargeTiers.filter(chargeTier => chargeTier['enxCPQ__Charge_Parent__r']
-                                             && chargeTier['enxCPQ__Charge_Parent__r']['enxCPQ__TECH_External_Id__c'] === charge['enxCPQ__TECH_External_Id__c'])
-                       .forEach(chargeTierToSave=>{ chargeTiersToSave.push(chargeTierToSave)});              
-
-            chargeToSave.chargeElements = chargeElementsToSave;
-            chargeToSave.chargeTier = chargeTiersToSave;
-            Util.writeFile('/charges/' + Util.sanitizeFileName(chargeName) + '.json', chargeToSave);
+            let chargeName = charge['Name'];
+            let chargeTechId = charge['enxCPQ__TECH_External_Id__c'];
+            let chargeFileName = chargeName + chargeTechId;
+
+            if(chargeFileNames.has(chargeFileName)){
```

Comment:
Is there way that we'll be having multiple charges with that same chargeFieldName? If so what do we do with them? Are they not important?